vue3-apollo-quasar/.storybook/preview.js

```diff
@@ -10,11 +10,23 @@ import 'quasar/dist/quasar.css';
 import { app } from '@storybook/vue3';
 import { Quasar } from 'quasar';
 import { createPinia } from 'pinia';
+import { initialize, mswDecorator } from 'msw-storybook-addon';
+
+import { setupGraphQL } from '@/init';
 
 const pinia = createPinia();
 
 app.use(Quasar, {}).use(pinia);
 
+// Initialize MSW
```

Comment:
Please add more useful info to this comment. What's MSW? Why are we using it?

```
MSW allows us to mock service workers...
```

Reply:
DOn't you think that will be cooler in the readme?
@AllanJeremy

Reply:
Added this to [the documentation PR](https://github.com/thisdot/starter.dev-github-showcases/pull/563)